OWNERS

```diff
@@ -1,7 +1,6 @@
 approvers:
   - bfournie
   - derekhiggins
-  - dhellmann
```

Comment:
Thanks @dhellmann for all your contributions!

/cc @dtantsur @zaneb @iurygregory @elfosardo @hardys